tests/component/test_scale_operations.py

```diff
@@ -0,0 +1,21 @@
+"""Test for validating different scale related operations."""
+import pytest
+
+import nidaqmx
+
+
+def test__polynomial__calcualte_reverse_coeff__returns_expected_reverse_coeff():
```

Comment:
The "then" clause always describes the expected behavior, so using the word "expected" is redundant.

Also, the "when" clause doesn't match the spelling of the function name.

```suggestion
def test__polynomial__calculate_reverse_poly_coeff__returns_reverse_coeff():
```

Reply:
updated as per suggestions